### DISCUSSION TITLE
[IPv6] Resolving doesnt work

### DISCUSSION THREAD

**Discussion by Operations8** [AdguardTeam/AdGuardHome#6741] [General]
I run docker on a debian 12 virtual machine. Debian host has got a static IPv6 ip (from the subnet from my ISP)
adguardhome docker is in bridge mode.

debian IP adressen:

2a**:****:****:20::245
172.*.*.245

nslookup
server 2a**:****:****:20::245
google.com

Gives me a time out

nslookup
server 172.*.*.245
google.com
 
Responds with a IPv4 address.

Setup Guide from adguardhome, shows me this:

127.0.0.1
::1
172.17.0.2
2001:db8:1::242:ac11:2

'
daemon.json:
{
  "ipv6": true,
  "fixed-cidr-v6": "2001:db8:1::/64",
  "experimental": true,
  "ip6tables": true
}
'
2001 = not my ISP subnet

When i look via Portainer to the adguardhome container network:

bridge | 172.17.0.2 | 172.17.0.1 
  | 2001:db8:1::242:ac11:2 | 2001:db8:1::1

my docker-compose file:
`
version: "2"

services:
  adguardhome:
    image: adguard/adguardhome
    network_mode: bridge
    container_name: adguardhome
    restart: unless-stopped
    volumes:
      - ./config:/opt/adguardhome/work
      - ./config:/opt/adguardhome/conf
      - /home/nick/NPM/letsencrypt:/opt/adguardhome/ssl
    ports:
      - 172.16.20.245:53:53/udp
      - 53:53/tcp
      - 784:784/udp
      - 853:853/tcp
      - 3333:3000/tcp
      - 99:80/tcp
      - 459:443/tcp
      
`

I have seen a lot of maybe solutions using macvlan. But i also sometimes read you should avoid using macvlan in docker. Since i am way more a windows guy, i am still learning and not sure if A is true of B is true. So in this cause i am trying to not use macvlan because as far as i can find what i have done should work.

Could you guys tell me why this is not working?

**Comment by amrmzr** [comment]
Debian has issues with ipv6 addresses, i solved mine yesterday by turning off ipv6.
Only ipv4 is on and now adguard home is working.
Just check in case what is listening on port 546
